Pad the central node data structure to 8 bytes, makes all the dirty
cast to pointer dereferences 8 byte aligned.

Index: source/fitz/list-device.c
--- source/fitz/list-device.c.orig
+++ source/fitz/list-device.c
@@ -109,6 +109,9 @@ typedef struct
 	unsigned int ctm    : 3;
 	unsigned int stroke : 1;
 	unsigned int flags  : 6;
+#if defined(__alpha__) || defined(__mips64__) || defined(__sparc64__)
+	uint32_t pad;
+#endif
 } fz_display_node;
 
 enum {
